### PR TITLE
support DualNlbFargateService

### PR DESCRIPTION
**Pull request by pahud**
Introduce `DualNlbFargateService` that allows you to create fargate services with externa/linternal NLBs.

At this moment, `scaleOnRequestCount` is not supported with NLB(see https://github.com/aws/aws-cdk/issues/6887). Let's see if there's any policy we can do in another PR.

Fixes: #78

### CODE REVIEW

File: test/default.test.ts [hunk 1]
```diff
@@ -760,3 +760,135 @@ test('fargate spot termination handler - partial spot', () => {
...
+  });
+
+  customerTask.addContainer('customer', {
+    image: ecs.ContainerImage.fromAsset(path.join(__dirname, '../services/golang/CommonService')),
+    portMappings: [
+      { containerPort: 8080 },
+    ],
+    environment: {
+      PRODUCT_SVC_URL: `${internalALBEndpoint}:9090`,
+      CUSTOMER_SVC_URL: `${internalALBEndpoint}:8080`,
```
Comment (neilkuan) [inline test/default.test.ts:800]:
```suggestion
      PRODUCT_SVC_URL: `${internalELBEndpoint}:9090`,
      CUSTOMER_SVC_URL: `${internalELBEndpoint}:8080`,
```

File: test/default.test.ts [hunk 2]
```diff
@@ -760,3 +760,135 @@ test('fargate spot termination handler - partial spot', () => {
...
+  });
+
+  productTask.addContainer('product', {
+    image: ecs.ContainerImage.fromAsset(path.join(__dirname, '../services/golang/CommonService')),
+    portMappings: [
+      { containerPort: 8080 },
+    ],
+    environment: {
+      PRODUCT_SVC_URL: `${internalALBEndpoint}:9090`,
+      CUSTOMER_SVC_URL: `${internalALBEndpoint}:8080`,
```
Comment (neilkuan) [inline test/default.test.ts:818]:
```suggestion
      PRODUCT_SVC_URL: `${internalELBEndpoint}:9090`,
      CUSTOMER_SVC_URL: `${internalELBEndpoint}:8080`,
```

File: test/default.test.ts [hunk 3]
```diff
@@ -760,3 +760,135 @@ test('fargate spot termination handler - partial spot', () => {
...
+          targetCpuUtilization: 50,
+        },
+      },
+      { task: customerTask, desiredCount: 2, internal: { port: 8080 } },
+      { task: productTask, desiredCount: 2, internal: { port: 9090 } },
+    ],
+    route53Ops: {
+      zoneName, // svc.local
+      externalElbRecordName: externalAlbRecordName, // external.svc.local
+      internalElbRecordName: internalAlbRecordName, // internal.svc.local
```
Comment (neilkuan) [inline test/default.test.ts:845]:
```suggestion
      externalElbRecordName: externalElbRecordName, // external.svc.local
      internalElbRecordName: internalElbRecordName, // internal.svc.local
```

**Review by neilkuan — CHANGES_REQUESTED** [review]
LGTM

Please help renew example code at README.md. thanks you. :)
```ts
new DualAlbFargateService(stack, 'Service', {
  spot: true, // FARGATE_SPOT only cluster
  tasks: [
    {
      task: orderTask,
      desiredCount: 2,
      internal: { port: 443, certificate },
      external: { port: 80 },
      // customize the service autoscaling policy
      scalingPolicy: {
        maxCapacity: 20,
        requestPerTarget: 1000,
        targetCpuUtilization: 50,
      },
    },
    { task: customerTask, desiredCount: 2, internal: { port: 8080 } },
    { task: productTask, desiredCount: 2, internal: { port: 9090 } },
  ],
  route53Ops: {
    zoneName: 'svc.local',
    externalAlbRecordName: 'external', -> externalElbRecordName
    internalAlbRecordName: 'internal', -> internalElbRecordName
  },
});
```

**Comment by pahud** [comment]
all resolved. please check again @neilkuan

**Review by neilkuan — APPROVED** [review]
LGTM

**Pull request merged**